play.pokemonshowdown.com/src/battle.ts

```diff
@@ -3393,6 +3396,32 @@ export class Battle {
 	}
 	runMajor(args: Args, kwArgs: KWArgs, preempt?: boolean) {
 		switch (args[0]) {
+		case 't:': {
+			if (this.realtime && this.timerCount) {
+				if (this.timeWait) {
+					setTimeout(() => {
```

Comment:
I don't see a check for fast forward. You should be skipping all timeouts when fast forwarding.

Reply:
Right, forgot about that. Accounted for it.